apps/website/src/data/presets/chicken.ts

```diff
@@ -0,0 +1,73 @@
+import { type StaticImageData } from "next/image";
+
+import backright from "@/assets/presets/chicken/backright.png";
+import down from "@/assets/presets/chicken/down.png";
+import downright from "@/assets/presets/chicken/downright.png";
+import food from "@/assets/presets/chicken/food.png";
+import home from "@/assets/presets/chicken/home.png";
+import left from "@/assets/presets/chicken/left.png";
+import oliversleep from "@/assets/presets/chicken/oliversleep.png";
+import right from "@/assets/presets/chicken/right.png";
+import window from "@/assets/presets/chicken/window.png";
+
+// import barrels from "@/assets/presets/chicken/barrels.png";
+// import nuggetsleep from "@/assets/presets/chicken/nuggetsleep.png";
+
+export interface Preset {
+  description: string;
+  image?: StaticImageData;
+}
```

Comment:
I wonder if this interface should be put in a dedicated file that can be referenced by all of these files, instead of being duplicated code in all of them?